defs/scenarios/storage/ceph/osd_flapping.yaml

```diff
@@ -0,0 +1,21 @@
+checks:
+  osd_flapping:
+    input:
+      path: var/log/ceph/ceph*.log
+    expr: '^([\d-]+)T([\d:]+\.\d)\S+ .+ wrongly marked me down at .+'
+conclusions:
+  osd-flapping:
+    decision: osd_flapping
+    raises:
+      type: hotsos.core.issues.CephOSDError
+      message: >-
+        OSD flapping occurred.
+        Causes include network issues (could do connectivity checks via ping,
+        netstat/ip for packet drops), disk problems (check smartctl, sar,
```

Comment:
You can actually check packet drops in a scenario - see https://github.com/canonical/hotsos/blob/main/defs/scenarios/storage/ceph/mon_elections_flapping.yaml